rest-openehr/src/main/java/org/ehrbase/rest/BaseController.java

```diff
@@ -113,15 +117,38 @@ public Map<String, Map<String, String>> add2MetaMap(
         return metaMap;
     }
 
+    /**
+     * This method is intended to return the base URI, however it is currently returning the request URI.
+     *
+     * @deprecated This method is deprecated and should not be used in favor of a more updated version.
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
Is there a reason why we cannot just remove it?

Reply:
removed